src/SmartComponents/Vulnerability/VulnerabilityCard.js

```diff
@@ -75,38 +75,36 @@ class VulnerabilityCard extends Component {
                     <TemplateCardHeader title="Vulnerabilities"/>
                 </TemplateCardHead>
                 <TemplateCardBody>
+                    <NumberDescription
+                        data={ vulnerabilitiesFetchStatus === 'fulfilled' && vulnerabilities.meta.total_items > 0 ?
+                            ` ${ vulnerabilities.meta.total_items } ` : '' }
+                        dataSize="md"
+                        linkDescription="CVEs impacting your systems"
```

Comment:
Need to build this string out in messages.js, gotta keep up with translations 🦹‍♀️